appveyor.yml

```diff
@@ -29,20 +27,22 @@ install:
       }
   - ps: if (-not (Test-Path $env:PYTHON)) { throw "No $env:PYTHON" }
   - echo "C:\Program Files\Microsoft SDKs\Windows\v7.1\Bin\SetEnv.cmd" /x64 > "C:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\bin\amd64\vcvars64.bat"
-  - pip install .[test]
+  - python -m pip install -U pip
+  - pip install -U setuptools wheel
+  - pip install -U -e .[test]
+
+matrix:
+  fast_finish: true
 
-build_script:
-  - pip install wheel
-  - python -W ignore setup.py -q bdist_wheel
+build: false
```

Comment:
Um, what???

If we're not building Windows wheels, why do we have `twine upload`?